Nb final edits overall (#25)
R4EPI/R4EPIs-website: content/_index.en.md

one of the community leaders in Ton-Habalan is waiting for his turn.* ***Source: Susanne Doettling/MSF, 2019.***

## R4epis - what is it?

**R4epis is a project to develop standardised data cleaning, analysis and
reporting tools to cover common types of outbreaks and population-based surveys
that would be conducted in an MSF emergency response setting.** 

This has been done through the development of the package **sitrep** in R software. The package provides 
field epidemiologists with **novel data management tools** as well as templates of
**automated "situation reports"** that cover outbreak investigations (acute jaundice syndrome, cholera, measles,
meningitis) and three of the MSF ERB pre-approved surveys (mortality, nutrition
and vaccination).

All of the report templates are contained within the **sitrep** package as RMarkdown templates that
can easily be used from within RStudio. The user then modifies a template to his/her needs.  

The templates address all aspects of:

* Data cleaning of outbreak linelists and survey data
* Analysis of data to report in terms of time, place, and person
* Analysis of survey data

> **All coding is open source and freely available and can be used by anyone.** For suggestions 

Reviewer: This relates to #18 

```suggestion
**All of the code is open source and freely available and can be used by anyone.** For suggestions 
```

Reviewer: Did you not like this suggestion?

> on what to add or change, please link to our [Github site](https://github.com/R4EPI) 

Reviewer: ```suggestion
on what to add or change, please [open an issue on GitHub](https://github.com/R4EPI/sitrep/issues).
```

or contact one of [our contributors](https://r4epis.netlify.com/credits/).

*Below: The "sitrep" package developed by R4Epis includes templates of automated situation reports* 

> ![Markdowns from R](images/SnipMarkdown.PNG?width=30pc)

Reviewer: "Markdowns" sounds weird to me. It's like calling huckleberries "huckles"

```suggestion
![Available situation report RMarkdown templates](images/SnipMarkdown.PNG?width=30pc)
```

	* Sampling strategy, sample size, design effect, and precision  
* [Training Materials](https://r4epis.netlify.com/training/)
	* [Basic R skills](https://r4epis.netlify.com/training/r_basics/)
	* [Case-study](https://r4epis.netlify.com/training/walk-through/) - a step-by-step example of using the sitrep templates 
* How to [contribute](https://r4epis.netlify.com/contributing/) to the R4Epis project
* [Frequently asked questions](https://r4epis.netlify.com/faq/)

## Who funded R4epis?

R4epis was funded through the [Sapling Nursery](https://www.msf.org.uk/sapling-nursery-grow-your-ideas), an MSF internal innovation fund for new projects that aim to improve the medical care MSF provides. 

## Who contributed to R4epis

R4epis has been a collaboration between the R for Epidemics Consortium ([RECON](https://www.repidemicsconsortium.org/)) and MSF. 

The project has merged the skills and experience of people in 
[coding in R software](https://blogs.msf.org/bloggers/r4epis-team/r4epis-hackathon), 
[field epidemiology](https://blogs.msf.org/bloggers/larissa/innovation-introducing-r4epis), 
spatial epidemiology, 
health information systems 
and public health.

> See [this page](https://r4epis.netlify.com/credits/) for a list of project contibutors. 

Reviewer: avoiding "this page" style links

```suggestion
Check out the [list of project contributors](https://r4epis.netlify.com/credits/). 
```

> See these MSF innovation blog posts describing [the origin](https://blogs.msf.org/bloggers/larissa/innovation-introducing-r4epis) 

Reviewer: ```suggestion
See these MSF innovation blog posts describing [the origin of the R4EPIs project](https://blogs.msf.org/bloggers/larissa/innovation-introducing-r4epis) 
```
